pkg/kube/inject/webhook.go

```diff
@@ -631,6 +631,11 @@ func applyMetadata(pod *corev1.Pod, injectedPodData corev1.Pod, req InjectionPar
 	if nw, ok := req.proxyEnvs["ISTIO_META_NETWORK"]; ok {
 		pod.Labels[label.TopologyNetwork.Name] = nw
 	}
+	if req.revision != "" {
```

Comment:
Can we put this in the injection templates instead?  we have been trying to move more logic to injections instead of in Go code. The only things remaining are things that are too hard to do in template

Reply:
This sounds like the template is just syntactically wrong. Likely because the `annotations` field is actually json-object syntax in the templates (`annotations: { ` - note the `{`)

Reply:
And we only set some annotations that we have no reasonable way top set in the template. We have 10 other annotations we set in the template and have been moving in that direction

Reply:
Ok, thanks! Since it's JSON, it was missing comma at the end of the line.